test/SILOptimizer/package-cmo-opaque-result.swift

```diff
@@ -0,0 +1,54 @@
+// RUN: %empty-directory(%t)
+// RUN: split-file %s %t
+
+// RUN: %target-build-swift %t/Lib.swift \
+// RUN: -module-name=Lib -package-name Pkg \
+// RUN: -parse-as-library -emit-module -emit-module-path %t/Lib.swiftmodule -I%t \
+// RUN: -Xfrontend -experimental-package-cmo -Xfrontend -experimental-allow-non-resilient-access \
+// RUN: -O -wmo -enable-library-evolution -Xfrontend -disable-availability-checking
```

Comment:
Why is -disable-availability-checking needed?

Reply:
No, this flag is used out of convenience, without it availability annotations would have to be used (because of opaque result types).